argnorm/normalizers.py

```diff
@@ -194,6 +172,11 @@ def preprocess_ref_genes(self, ref_genes):
 
 class AbricateNormalizer(BaseNormalizer):
     def __init__(self, database=None, is_hamronized=False) -> None:
+        try:
+            database in ['ncbi', 'deeparg', 'resfinder', 'sarg', 'megares', 'argannot']
```

Comment:
This doesn't do anything. SHould be

```python
if database not in ...:
   raise ...
```

Was there a test for this functionality?